components/ProductCard/index.js

```diff
@@ -3,14 +3,19 @@ import styled from "styled-components";
 import { formatToEUR } from "../../helper/formatToEUR";
 import { useRouter } from "next/router";
 import useStore from "../../hooks/useStore";
+import "chart.js/auto";
+import { Bar } from "react-chartjs-2";
+import { chartOptions } from "../../helper/chartOptions";
+import { createChartData } from "../../helper/createChartData";
+import useMedia from "../../hooks/useMedia";
+import { sumTotalPrice } from "../../helper/sumTotalPrice";
 
 export default function ProductCard({ project, editState }) {
   const router = useRouter();
   const [moveToActive] = useStore((state) => [state.moveToActive]);
-  const sumUpArray = (accumulator, currentValue) => accumulator + currentValue;
-  const totalPrice = project?.items
-    .map((item) => Number(item.price))
-    .reduce(sumUpArray, 0);
+  const labelColor = useMedia(["(prefers-color-scheme: dark)"], [true], false)
```

Comment:
In a future step, do you plan to set the `[true]` and `false` arguments programmatically to the opposite so that the `labelColor` changes on user input?
I just don't understand why you calculate this value with `useMedia` to check it to simply set the color to `white` or `black`. Is it even possible to get the `black` value (or the `white `, I'm not sure what the normal result is).

Reply:
i will implment a simpler hook, thx.